xml/System.Security.Cryptography.Xml/CryptoSignedXmlRecursionException.xml

```diff
@@ -24,7 +24,9 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>This exception helps catch the signed XML recursion limit error.
+            This is being caught in the SignedXml class while computing the hash. ComputeHash can throw different kind of exceptions.
+            This unique exception helps catch the recursion limit issue.</summary>
```

Comment:
```suggestion
```

Reply:
Not sure about removing this. @bartonjs?

Reply:
@carlossanlop It's already stated on line 27, so it was a repeat.

Reply:
Removing the line LGTM, especially since currently there are two closing summary tags.

Reply:
@gewarren ah sorry, didn't notice that.